spec/models/comment_spec.rb

```diff
@@ -0,0 +1,9 @@
+require "rails_helper"
+
+describe Comment do
+  let!(:comment) { FactoryGirl.create(:comment) }
+  it "sends an email" do
+    expect { comment.notification_mailer 
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.<br>Trailing whitespace detected.